Testing/VelaptorTests/Content/ContentLoaderTests.cs

```diff
@@ -61,10 +55,10 @@ public void Load_WhenLoadingSounds_LoadsSound()
             var loader = CreateContentLoader();
 
```

Comment:
Refactor method name 
```suggestion
        public void LoadSound_WhenLoadingSounds_LoadsSound()
```